Kinvey/Kinvey/Error.swift

```diff
@@ -89,6 +89,14 @@ public enum Error: Swift.Error, CustomStringConvertible, CustomDebugStringConver
         }
     }
     
+    public var errorDescription: String? {
+        return description
+    }
+    
+    public var failureReason: String? {
+        return description
```

Comment:
right! so in this case we should actually invert the order... `errorDescription` should have a bigger explanation and `failureReason` a short version containing only the reason